static/css/style.css

```diff
@@ -62,3 +62,5 @@ a img {
         background-position: top center !important;
     } 
 }
+
+.DocSearch-Modal { top: 27px;}
```

Comment:
NIT: new line

Reply:
Where? At the end?

Reply:
yeah. github shows  a red circle when that's missing.